### PR TITLE
[RM CH05 ] Secuirty section "Add Link to Chapter to CH07"

### PR DESCRIPTION
**Pull request by ASawwaf**
[RM CH05] 5.1.4 Security #1669

### CODE REVIEW

File: doc/ref_model/chapters/chapter05.md [hunk 1]
```diff
@@ -111,7 +111,8 @@ The following sections detail the Cloud Infrastructure Software Profile features
 
 <a name="5.1.4"></a>
 ### 5.1.4 Security
-_**Comment:** To be worked on._
+_**Comment:** All Secuirty features defined in RM Chapter 07
```
Comment (deebhatia) [inline doc/ref_model/chapters/chapter05.md:114]: lacks a trailing underscore.
nit: Secuirty -> Security

Comment (pgoyal01) [reply doc/ref_model/chapters/chapter05.md:114]:
```suggestion
_**Comment:**_  RM Chapter 07 (./chapter07.md) discusses and specifies requirements for platform security.
```

File: doc/ref_model/chapters/chapter05.md [hunk 2]
```diff
@@ -111,7 +111,8 @@ The following sections detail the Cloud Infrastructure Software Profile features
 
 <a name="5.1.4"></a>
 ### 5.1.4 Security
-_**Comment:** To be worked on._
+_**Comment:** All Secuirty features defined in RM Chapter 07 
+https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter07.md
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter05.md:115]: delete as included in suggested text in sentence before

**Comment by rabi-abdel** [comment]
@ASawwaf can you please address comments above?

**Comment by ASawwaf** [comment]
@pgoyal01 @deebhatia , is there any pending point from my side ?

**Comment by deebhatia** [comment]
> @pgoyal01 @deebhatia , is there any pending point from my side ?

@ASawwaf Yes. PR comments are pending. https://github.com/cntt-n/CNTT/pull/1670#discussion_r440193660 chain. Also, PR has  conflicts. Please resolve.

**Comment by ASawwaf** [comment]
@deebhatia Done thx

**Comment by deebhatia** [comment]
@ASawwaf Checking in master branch, it turns out similar change is already done in commit
https://github.com/cntt-n/CNTT/commit/7457386aa5e9f7202b9b9e59f2f62a41c54a5a41
If the merged commit's language looks ok, then this PR can be closed without merging.

**Comment by ASawwaf** [comment]
noted

**Pull request closed without merging**